tests/operation_builder_test.py

```diff
@@ -419,7 +419,7 @@ class VarSBRegionOp(Operation):
     regs = VarSingleBlockRegionDef()
 
 
-def test_opt_sbregion_one_block():
```

Comment:
Pylance reports the naming of this test as an issue? Am I missing something? Is this doubly-defined or something? O/w it feels a bit crazy to report this.
I do agree with the change though.